report/latest_map.go

```diff
@@ -178,14 +177,6 @@ func (m LatestMap) toIntermediate() map[string]LatestEntry {
 	return intermediate
 }
 
-func (m LatestMap) fromIntermediate(in map[string]LatestEntry) LatestMap {
-	out := ps.NewMap()
-	for k, v := range in {
-		out = out.Set(k, v)
-	}
-	return LatestMap{out}
-}
-
 // CodecEncodeSelf implements codec.Selfer
 func (m *LatestMap) CodecEncodeSelf(encoder *codec.Encoder) {
 	if m.Map != nil {
```

Comment:
> I thought we could do the encoding in a backwards compatible way. Can't we?

We can, but then its gets harder to detect when we break backwards compatibility.